lib/crypto.js

```diff
@@ -647,9 +647,21 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
-exports.randomBytes = exports.pseudoRandomBytes = randomBytes;
-
-exports.rng = exports.prng = randomBytes;
+exports.randomBytes = util._deprecateSync(randomBytes,
```

Comment:
Instead of introducing a new deprecation method for sync methods, you can do something like this

```js
diff --git a/lib/crypto.js b/lib/crypto.js
index bfe7837..5cd3bcd 100644
--- a/lib/crypto.js
+++ b/lib/crypto.js
@@ -647,7 +647,14 @@ exports.setEngine = function setEngine(id, flags) {
   return binding.setEngine(id, flags);
 };
 
-exports.randomBytes = exports.pseudoRandomBytes = randomBytes;
+exports.randomBytes = exports.pseudoRandomBytes = function(size, callback) {
+    const warning = 'randomBytes used as async is deprecated. ' +
+                    'Use randomBytesSync instead.';
+    if (typeof callback === 'function') {
+        internalUtil.printDeprecationMessage(warning);
+    }
+    return randomBytes(size, callback);
+}
 
 exports.rng = exports.prng = randomBytes;
 ```